app/schema_validation/service_join_request.py

```diff
@@ -10,3 +10,39 @@
     "required": ["requester_id", "contacted_user_ids"],
     "additionalProperties": False,
 }
+
+service_join_request_update_schema = {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "description": "Schema for updating a service join request",
+    "type": "object",
+    "properties": {
+        "permissions": {
+            "type": "array",
+            "items": {
+                "type": "string",
+                "enum": [
+                    "manage_users",
+                    "manage_templates",
+                    "manage_settings",
+                    "send_texts",
+                    "send_emails",
+                    "send_letters",
+                    "manage_api_keys",
+                    "platform_admin",
+                    "view_activity",
+                ],
+            },
+            "description": "List of permissions being granted or modified",
+        },
+        "status_changed_by_id": {"type": "string", "format": "uuid"},
+        "status": {
+            "type": "string",
+            "enum": ["pending", "approved", "rejected"],
```

Comment:
Looks like `cancelled` [has been added](https://github.com/alphagov/notifications-api/pull/4221) as an option since this was originally written.